doc/refman/RefMan-pre.tex

```diff
@@ -1335,6 +1335,157 @@ \section*{Credits: version 8.7}
+  result can be seen at ???
+\item Tools: experimental ``-mangle-names'' option to coqtop/coqc for
+  linting proof scripts, by Jasper Hugunin.
+\end{itemize}
+
+On the implementation side, the \texttt{dev/doc/changes.md} file
+documents the numerous changes to the implementation and improvements of
+interfaces. The file provides guidelines on porting a plugin to the new
+version.
+
+Version 8.8 also comes with a bunch of smaller-scale changes and
+improvements regarding the different components of the system. They are
+documented in the \texttt{CHANGES} file.
+
+The efficiency of the whole system has seen improvements thanks to
+contributions from Gaëtan Gilbert, Pierre-Marie Pédrot, Maxime Dénès and
+Matthieu Sozeau and performance issue tracking by Jason Gross and Paul
+Steckler.
+
+The official wiki and the bugtracker of {\Coq} migrated to the
+github platform, thanks to the work of Pierre
+Letouzey and Théo Zimmermann. Gaëtan Gilbert, Emilio Jesús Gallego Arias
+and ... worked on maintaining and improving the continuous integration
+system.
+
+The OPAM repository for {\Coq} packages has been maintained by Guillaume
+Melquiond, Matthieu Sozeau, Enrico Tassi with contributions from many
+users. A list of packages is available at \url{https://coq.inria.fr/opam/www/}.
+
+TODO Outdated?
+Packaging tools and software development kits were prepared by Michael
+Soegtrop with the help of Maxime Dénès and Enrico Tassi for Windows, and
+Maxime Dénès for MacOS X. Packages are regularly built on the
+Travis and GitLab continuous integration server.
+
+The 40 contributors for this version are Yves Bertot, Joachim
+Breitner, Tej Chajed, Arthur Charguéraud, Jacques-Pascal Deplaix, Maxime
+Dénès, Jim Fehrle, Yannick Forster, Gaëtan Gilbert, Jason Gross, Samuel
+Gruetter, Thomas Hebb, Hugo Herbelin, Jasper Hugunin, Emilio Jesus
+Gallego Arias, Ralf Jung, Johannes Kloos, Matej Košík, Robbert Krebbers,
+Tony Beta Lambda, Vincent Laporte, Pierre Letouzey, Farzon Lotfi,
+Cyprien Mangin, Guillaume Melquiond, Raphaël Monat, Carl Patenaude
+Poulin, Pierre-Marie Pédrot, Matthew Ryan, Matt Quinn, Sigurd Schneider,
+Bernhard Schommer, Matthieu Sozeau, Arnaud Spiwack, Paul Steckler,
+Enrico Tassi, Anton Trunov, Martin Vassor, Vadim Zaliva and Théo
+Zimmermann.
+
+The development process was coordinated by Matthieu Sozeau. Maxime Dénès
+was in charge of the release process. The maintainer of this release
+is ???.
+
+Many power users helped to improve the design of the new features via
+the bug tracker, the pull request system, the {\Coq} development mailing
+list or the coq-club mailing list. Special thanks to the users who
+contributed patches and intensive brain-storming and code reviews,
+starting with Jason Gross, Ralf Jung, Robbert Krebbers, Amin Timany,
+..???.  It would however be impossible to mention exhaustively the names
+of everybody who to some extent influenced the development.
+
+Version 8.8 is the third release of {\Coq} developed on a time-based
```

Comment:
From here down to the bottom I think this is more part of a release announcement rather than a credits section.

Reply:
Right, I agree